packages/gguf/package.json

```diff
@@ -32,6 +32,7 @@
 		"format:check": "prettier --check .",
 		"prepublishOnly": "pnpm run build",
 		"build": "tsup src/index.ts --format cjs,esm --clean --dts",
```

Comment:
```suggestion
		"build": "pnpm build:llm && tsup src/index.ts --format cjs,esm --clean --dts",
```

`build:llm` should be called here before build right? (cc: @coyotte508)

Reply:
or maybe run it once a week from a GitHub Action or something?

Reply:
It fetches a remote file so yes - calling it with a cron job is better than at every install

Reply:
I prefer to have it run manually, because this way we can verify the generated script before pushing. But if we can run a Github Action workflow and create new PR if there're changes, it's also a good idea.